agent/tool-scripts/base-tool

```diff
@@ -711,19 +710,21 @@ start)
 
 	if [[ ! -z "${options}" ]]; then
 		# Record any options for later post-processing use.
-		echo "${options}" > ${tool_output_dir}/${tool_name}.options
+		echo "${options}" > ./${tool_name}.options
```

Comment:
Well, I changed it so that I can test it easily by hand.  One should be able to do:

```
cd /var/tmp/
mkdir test
tool-scripts/turbostat --start --dir=./test
```

But relative paths were, as they say, always relative.  So `base-tool` would look to write files in `/var/tmp/test/turbostat/test/turbostat`.